src/Catel.MVVM/Catel.MVVM.Shared/Collections/SuspensionContext.cs

```diff
@@ -35,167 +26,46 @@ public class SuspensionContext<T>
         /// <param name="mode">The suspension mode.</param>
         public SuspensionContext(SuspensionMode mode)
         {
-            _suspensionMode = mode;
+            Mode = mode;
         }
-        #endregion
+        #endregion Constructors
 
         #region Properties
         /// <summary>
-        /// Gets or sets the suspension count.
-        /// </summary>
-        public int Count
-        {
-            get
-            {
-                return _suspensionCount;
-            }
-
-            set
-            {
-                if (value != _suspensionCount)
-                {
-                    if (value < 0)
-                    {
-                        _suspensionCount = 0;
-                    }
-                    else
-                    {
-                        _suspensionCount = value;
-                    }
-                }
-            }
-        }
-
-        /// <summary>
-        /// Gets the suspension mode.
-        /// </summary>
-        public SuspensionMode Mode
-        {
-            get
-            {
-                return _suspensionMode;
-            }
-        }
-
-        /// <summary>
-        /// Gets the indices od the added items while change notifications were suspended.
-        /// </summary>
-        public List<int> NewItemIndices
```

Comment:
Only FastBindingList is using this. So it was moved to ExtendedSuspensionContext.